modEvtMgrImpl/test/org/aion/evtmgr/impl/abs/AbstractEventMgrTest.java

```diff
@@ -34,8 +34,7 @@
 import static junit.framework.TestCase.assertEquals;
 
 public class AbstractEventMgrTest {
-    private Properties properties = new Properties();
```

Comment:
This class should to be renamed as well since `AbstractEventMgr.class` -> `EventManager.class`